### PR TITLE
410 rss cell bug

### PR DESCRIPTION
**Pull request by w-klijn**
The rss cell had a single time variable per cell that was set per cell group.
It was standard set to epoch at end of the epoch.

This causes problems with spike time steps bigger then epoch and when there are cells with different settings as observed by Avleen.

Each cell now has an unique time variable.
Change includes a test that went from red to green after the change.

### CODE REVIEW

**Comment by w-klijn** [comment]
Intersting, this problem will be present with basically all time stepped neurons. 
I will also change this on the ipss_cell. I already use they approach in some corned cases. Thus extending this to the whole cell should be easy

**Comment by bcumming** [comment]
You are right, it is a problem for many neurons that we time step. @halfflat and I decided that it would be best to have the "expected" behavior whenever it is reasonable to implement.

File: src/rss_cell_group.hpp [hunk 1]
```diff
@@ -38,12 +38,17 @@ class rss_cell_group: public cell_group {
 
     void advance(epoch ep, time_type dt, const event_lane_subrange& events) override {
         for (auto& cell: cells_) {
-            auto t = std::max(cell.start_time, cell.t);
+            auto t_start = std::max(cell.start_time, cell.t);
             auto t_end = std::min(cell.stop_time, ep.tfinal);
 
+            std::size_t step = 0;
```
Comment (bcumming) [inline src/rss_cell_group.hpp]: To have full reproducable sequences, use `t = cell.start_time+cell.step*cell.period`, and increment `cell.step` (and set it to 0 when resetting the cell).

Comment (w-klijn) [reply src/rss_cell_group.hpp]: Done

**Pull request merged**